src/I18n/Number.php

```diff
@@ -296,6 +342,8 @@ public static function formatter($options = [])
             $type = $options['type'];
             if ($options['type'] === static::FORMAT_CURRENCY) {
                 $type = NumberFormatter::CURRENCY;
+            } elseif ($options['type'] === static::FORMAT_CURRENCY_ACCOUNTING) {
+                $type = static::CURRENCY_ACCOUNTING;
```

Comment:
Isnt that FORMAT_CURRENCY_ACCOUNTING?
You need to add test cases for this specific one, also please consider new features going to 4.x first, and 3.next (3.9) as following backport please.

Reply:
This is currently targeting master but also adding new features. Could you change this pull request to target 3.next instead?